corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -177,6 +178,12 @@ def property_changed_in_action(case_transaction, case_id, case_property_name):
     from casexml.apps.case.xform import get_case_updates
     PropertyChangedInfo = namedtuple("PropertyChangedInfo", 'transaction new_value modified_on')
     include_create_fields = case_property_name in ['owner_id', 'name', 'external_id']
+
+    if not should_use_sql_backend(case_transaction.form.domain):
+        # couch domains return 2 transactions for case properties created in a create form
+        if case_transaction.is_case_create and not include_create_fields:
```

Comment:
By the names, it seems like this is something that shouldn't happen? Is this comment saying that for couch domains another transaction will add these fields?

Reply:
yeah, this is something that surprised me, and seems to be inconsistent between SQL and Couch. 

In SQL, when you create a case with a bunch of case properties you only get a single `CaseTransaction` - A "create" transaction, which contains a `create` action (which creates things like the name and owner_id) and an `update` action (which creates the case properties).

In couch, when you create a case with a bunch of case properties, you get 2 `CommCareCaseAction`s - A "create" which has both a `create` and `update` actions, and an "update" which only has an `update`.